api/README.md

```
@@ -301,16 +301,134 @@ docker build -t bocchi-api:latest .
 docker buildx build --platform linux/amd64,linux/arm64 .
 ```
 
+### Docker Development
+
+#### Local Docker Build
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)